addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/InformationDisclosureSuspiciousComments.java

```diff
@@ -125,24 +128,29 @@ private void raiseAlert(HttpMessage msg, int id, String detail) {
         parent.raiseAlert(id, alert);
     }
 
-    private List<Pattern> getPatterns() {
-        if (patterns == null) {
-            patterns = new ArrayList<>();
-            String line = null;
-            File f = new File(Constant.getZapHome() + File.separator + databaseErrorFile);
+    private List<Pattern> setPatterns() {
```

Comment:
`set` does not seem right, it's not setting anything, just returning (`get`/`load`/`read` would be better IMO).